internal/core/onlinecheck/onlinecheck_test.go

```diff
@@ -39,7 +39,7 @@ func TestRefreshOnlinePlayers_Success(t *testing.T) {
 	a := &Adapter{
 		api:            mockAPI,
 		guildIdToWorld: cmap.New[string](),
-		players:        cmap.New[[]string](),
+		players:        cmap.New[map[string]struct{}](),
```

Comment:
This change is good and I'd keep it. Not sure about forcing lowercase everywhere though.